packages/core/lib/commands/init/initSource/truffle-config.js

```diff
@@ -93,13 +93,22 @@ module.exports = {
     }
   },
 
-  // Truffle DB is currently disabled by default; to enable it, change enabled: false to enabled: true
+  // Truffle DB is currently disabled by default; to enable it, change enabled:
+  // false to enabled: true. The default storage location can also be
+  // overridden by specifying the adapter settings, as shown in the commented code below.
   //
-  // Note: if you migrated your contracts prior to enabling this field in your Truffle project and want
-  // those previously migrated contracts available in the .db directory, you will need to run the following:
+  // Note: your previously migrated contracts will not be available in
+  // truffle-db. We recommnd you backup your artifacts before utilizing Truffle
+  // DB for the first time with:
   // $ truffle migrate --reset --compile-all
```

Comment:
Note that this won't "back up" anything! This will destroy the backups.

Reply:
Oh hahaha I can't believe I missed that! Come to think of it, the previously migrated contracts *would* still be available via artifacts, right? Just not in `db`? What should we suggest to people in that situation? This may be a bit more nuanced than I originally thought.

Reply:
I read that as back up your artifacts --stop-- before moving on to using truffle db. I see the ambiguity now.